### PR TITLE
Get plain text database responses

### LINKED ISSUE #1: DBLP changed their interface
Instead of
```
http://dblp.uni-trier.de/rec/bibtex1/journals/cacm/Lipton75.bib
```
it is now
```
http://dblp.uni-trier.de/rec/journals/cacm/Lipton75.bib?param=1
```

Bibly would need to support placeholders in the configuration file, in order to specify where a key should go in the URL instead of just appending it. However, even after hard-coding this for DBLP the downloads still fail. For some reason `fetchRecord` always returns the empty string.

### PR DESCRIPTION
**Pull request by michael-emmi**
* Fixes #1

### CODE REVIEW

**Comment by beillahi** [comment]
I also had the same problem as in #1 

I was able to fix it by just having:
 ```"url": "http://dblp.uni-trier.de/rec/bibtex"``` 
instead of 
```"url": "http://dblp.uni-trier.de/rec/bibtex1"```

(removed 1 at the end of the url)

Maybe it would be nice if you updated also the url in [here](https://github.com/michael-emmi/bibly/blob/master/README.md#usage)

File: bin/cli.js [hunk 1]
```diff
@@ -37,6 +37,8 @@ let cli = meow(`
     console.log(`warning: could not read bibly.json`);
   }
 
+  console.log(cli.flags);
```
Comment (bkragl) [inline bin/cli.js]: Change to debug output?

**Review by bkragl — APPROVED** [review]
Nice 👍

My `bibly.json` file now looks as follows (maybe this can be documented in `README.md`):
```
{
  "latex": "main.tex",
  "databases": {
    "DBLP": {
      "url": "http://dblp.uni-trier.de/rec",
      "suffix": ".bib",
      "parameters": {"param": "1"},
      "file": "dblp.bib"
    }
  }
}
```

One small suggestion in the code below.

File: bin/cli.js (hunk 1 above)
Comment (michael-emmi) [reply bin/cli.js]: Removing for now

**Comment by michael-emmi** [comment]
@beillahi @bkragl updated README.md and removed leftover console output.

**Pull request merged**